tools/sapp/sapp/interactive.py

```diff
@@ -1409,7 +1412,8 @@ def _create_issue_output_string(
                 f" Sources: {sources_output if sources_output else 'No sources'}",
                 f"   Sinks: {sinks_output if sinks_output else 'No sinks'}",
                 (f"Location: {issue.filename}" f":{issue.location}"),
-            ]
+                f"Min_trace_length_to_sources: {issue.min_trace_length_to_sources}",
+                f"Min_trace_length_to_sinks: {issue.min_trace_length_to_sinks}",            ]
```

Comment:
Looks like this is making `black` (or formatter) unhappy. You can see the errors [here](https://travis-ci.com/facebook/pyre-check/jobs/275945901). Do you mind running `black` on this file (or just manually updating the three issues it found)
```suggestion
                f"Min_trace_length_to_sinks: {issue.min_trace_length_to_sinks}",
            ]
```